tests/cypress/e2e/all/header.cy.ts

```diff
@@ -102,7 +102,7 @@ describe('Header', () => {
     cy.get(`[data-testid='chain-icon-${eth}']`).should('be.visible')
     cy.get(`[data-testid='btn-change-chain']`).click()
     cy.get(`[data-testid='menu-item-chain-${arbitrum}']`).click()
-    cy.get(`[data-testid^='menu-item-chain-']`).should('not.be.visible')
+    cy.get(`[data-testid^='menu-item-chain-']`).should('not.exist')
```

Comment:
Heh I think you've also fixed this in your network switching PR. Can stay, all fine.